File: design-patterns/go/bridge/ui_element/table.go
// table.go

package main

import "fmt"

type Table struct {
	colorSchema ColorSchema
}

func NewTable(colorSchema ColorSchema) (table *Table) {
	table = &Table{}
	table.colorSchema = colorSchema
	return
}
func (table *Table) PrintElement() {
	table.colorSchema.SetColor()

	fmt.Println("Printing Table")
}

File: design-patterns/php/bridge/uielement/Table.php
<?php
// Table.php

namespace BigBoxCode\DesignPattern\Bridge\UiElement;


class Table extends UIElement {
    public function __construct(Color $color) {
        parent::__construct($color);
    }

    public function printElement(): void {
        $this->color->setColor();

        echo "Printing Table\n";
    }
}
